script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -220,7 +220,7 @@ Oh, I have let most of the servants go, but I have left one for each of us, so t
 "No, that's way too much! If you go around doing things like that, you'll be brought up for questioning at the next family reunion!"
 <Page63>
 "Please stop blathering on so much.//「もう、つべこべ言わないでください。
-Would you not feel more comfortable if we were the only ones in the mansion instead of it being teeming of people?"//　兄さんだって屋敷の中に人が溢れかえっているより、私たちしかいないほうが気持ちは楽でしょう？」
+Would you not feel more comfortable if we were the only ones in the mansion instead of it being teeming with people?"//　兄さんだって屋敷の中に人が溢れかえっているより、私たちしかいないほうが気持ちは楽でしょう？」
```

Comment:
can we take out being?

Reply:
after reading this line for the one trilioth time, I've decided this is the wrong structure in english. "if we were the only" sounds like its a situation she's striving for rather than the one they're in.
> Would you feel more comfortable if the mansion were teeming with people, rather than it just being the two of us?

Reply:
I feel like that edit makes it more of a question than it is.
> Surely you would agree that it is far better to have the mansion to ourselves than for it to be teeming with other people?